### PR TITLE
Add duration assigment step and create task on save

### PR DESCRIPTION
**Pull request by codetheweb**
Closes https://github.com/sfbrigade/compass/issues/238.

Demo:

https://github.com/sfbrigade/compass/assets/7410405/2011b05c-7ca4-4e18-8b08-31b1457d62d4

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -79,16 +81,18 @@ export default function App({
...
-        <QueryClientProvider client={queryClient}>
-          <SessionProvider session={pageProps.session}>
-            {errorMessage && <CustomToast errorMessage={errorMessage} />}
-            <Layout>
-              <Component {...pageProps} showErrorToast={toast.error} />
-            </Layout>
-          </SessionProvider>
-        </QueryClientProvider>
-      </trpc.Provider>
+      <LocalizationProvider dateAdapter={AdapterDateFns}>
```
Comment (tessathornberry) [inline src/pages/_app.tsx:84]: Thanks for doing this!

**Review by tessathornberry — APPROVED** [review]
Great work! I only had questions on the maxwidth, but will talk to the design systems folks about how they want their long selection labels to look at that size/in mobile:
image

**Comment by tessathornberry** [comment]
FYI, @codetheweb ,I want to be sure we include the tests and CSS for #238 before we consider it closed. Did you want to edit your initial submit comment, or should we proceed with this and do those two issue separately?

**Comment by codetheweb** [comment]
> Did you want to edit your initial submit comment, or should we proceed with this and do those two issue separately?

Let's make two new issues for that.

**Pull request merged**